static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.spec.tsx

```diff
@@ -1724,9 +1703,7 @@ describe('WidgetBuilder', function () {
           orgFeatures: [...defaultOrgFeatures],
         });
 
-        await screen.findByText('Group your results');
```

Comment:
don't we need to check if this label is available in the UI?

Reply:
There was no `expect` here, so if test below passes, I think that should be fine.